discovery-provider/plugins/notifications/src/processNotifications/mappers/userNotificationSettings.ts

```diff
@@ -410,6 +406,20 @@ export class UserNotificationSettings {
     )
     return userBrowserSettings
   }
+
+  async getUserNotificationBrowsers(userId: number): Promise<WebPush[]> {
```

Comment:
amazing! would you mind adding tests for this in `userNotificationSettings.test.ts`?